qemu/tests/cfg/virtio_trace_pipenb.cfg

```diff
@@ -0,0 +1,7 @@
+- virtio_trace_pipenb:
+    only Linux
+    type = virtio_trace_pipenb
+    serials += " vs2"
+    serial_type_vs2 = virtserialport
+    chardev_backend_vs2 = pipe
+    auto_create_pipe = yes
```

Comment:
we can delete this since default value is 'yes' : 
`params.get('auto_create_pipe', 'yes') == "yes")`